src/AmpWpPlugin.php

```diff
@@ -78,6 +79,7 @@ protected function get_service_classes() {
 			'rest.options_controller'          => OptionsRESTController::class,
 			'server_timing'                    => ServerTiming::class,
 			'obsolete_block_attribute_remover' => ObsoleteBlockAttributeRemover::class,
+			'amo_validated_url_stylesheet_gc'  => ValidatedUrlStylesheetDataGarbageCollection::class,
```

Comment:
Typo in identifier. But also, I don't think we need prefixing here.
```suggestion
			'validated_url_stylesheet_gc'      => ValidatedUrlStylesheetDataGarbageCollection::class,
```